pallets/committee/src/lib.rs

```diff
@@ -76,10 +76,22 @@ pub mod pallet {
         type MinCouncilVotes: Get<usize>;
 
         /// Origin that is permitted to create proposals
-        type ProposalSubmissionOrigin: EnsureOrigin<<Self as frame_system::Config>::Origin>;
+        type ProposalSubmissionOrigin: EnsureOrigin<
+            <Self as frame_system::Config>::Origin,
+            Success = <Self as frame_system::Config>::AccountId,
+        >;
 
         /// Origin that is permitted to execute approved proposals
-        type ProposalExecutionOrigin: EnsureOrigin<<Self as frame_system::Config>::Origin>;
+        type ProposalExecutionOrigin: EnsureOrigin<
+            <Self as frame_system::Config>::Origin,
+            Success = <Self as frame_system::Config>::AccountId,
+        >;
+
+        /// Origin that is permitted to vote
+        type ProposalVoteOrigin: EnsureOrigin<
+            <Self as frame_system::Config>::Origin,
+            Success = <Self as frame_system::Config>::AccountId,
+        >;
```

Comment:
what's the reasoning behind this extra origin? Aren't only member's allowed to vote?